owncloudComLibrary/src/main/java/com/owncloud/android/lib/resources/appregistry/GetRemoteAppRegistryOperation.kt

```diff
@@ -75,4 +74,19 @@ class GetRemoteAppRegistryOperation(private val appUrl: String?) : RemoteOperati
 
         return result
     }
+
+    private fun removeSubfolder(url: String): String {
+        val doubleSlashIndex = url.indexOf("//")
```

Comment:
Why `//`? What is useful for checking if the URL contains `//`? I don't understand the rest of the algorithm though

Reply:
The algorithm is formed so that if it finds a slash `/` it deletes everything in front of it, thus eliminating the subfolders. But the algorithm can detect these slashes // and will think that it has to remove the content in front since it has found a slash. Therefore, it must be validated that if there are two `//` slashes in a row, it does not delete the content afterwards.
In a more practical way this would be the result in each case:
Sent URL:
https://test.issue.com
Returned URL:
https://test.issue.com

URL with subfolder:
https://test.issue.com/folder1/folder2
Returned URL:
https://test.issue.com

By the way, I changed a little bit the logic because Base URL is always supposed to have `https://` or `htpp://`.

Reply:
The returned URLs in your comment are not correct, they must finish in `/`. But the code is correct, so let's resolve this